CHANGELOG.md

```diff
@@ -0,0 +1,163 @@
+Bindlib 6.0 (upcoming)
+======================
+
+Breaking changes:
+- Major change of semantics that impacts variable renaming (and printing). The
+  new semantics is explained in the documentation (see the `.mli` file), and a
+  guide for porting code is provided below.
+
+Other changes:
+- Compilation using Dune.
+- Generation of the `bindlib.opam` file using Dune.
+- Require at least OCaml 4.07.0 (not strictly necessary, but solves CI issues
+  with the installation of Dune on older versions).
+- Cleaning up of the documentation in the code (`.ml` file).
+- Polishing of the interface documentation (`.mli` file).
+- Add the `unbind2_in` function (similar to `unbind2` but with a context).
+- Add the `unmbind2_in` function (similar to `unmbind2` but with a context).
+- Add a functor `Ctxt` that can be used to define a custom variable renaming
+  policy, given a type of contexts and a few configurations.
+
+This releases breaks your printing code: here is how to fix it
+--------------------------------------------------------------
+
+TODO
```

Comment:
We still need to write a guide on how to port code to this new version of Bindlib.

Reply:
C'est fait, tu peux jeter un œil @craff?